src/CalcViewModel/Common/TraceLogger.h

```diff
@@ -83,12 +83,12 @@ namespace CalculatorApp::ViewModel::Common
         void LogVariableSettingsChanged(Platform::String ^ setting);
         void LogGraphSettingsChanged(GraphSettingsType settingsType, Platform::String ^ settingValue);
         void LogGraphTheme(Platform::String ^ graphTheme);
+        void LogInputPasted(CalculatorApp::ViewModel::Common::ViewMode mode);
+        void LogPlatformExceptionInfo(CalculatorApp::ViewModel::Common::ViewMode mode, Platform::String ^ functionName, Platform::String ^ message, int hresult);
 
         internal:
-        // CSHARP_MIGRATION: TODO:
         void LogPlatformException(CalculatorApp::ViewModel::Common::ViewMode mode, Platform::String ^ functionName, Platform::Exception ^ e);
```

Comment:
What's the difference between this one and `LogPlatformExceptionInfo`?

Reply:
`LogPlatformExceptionInfo` only cares about message and hresult of a exception.
Though `LogPlatformException` seems to be handling more details than `LogPlatformExceptionInfo`, it does the same work.
The way of introducing `LogPlatformExceptionInfo` is because `Platform::Exception` doesn't across the ABI boundary between C# and C++/CX.

Reply:
> Platform::Exception doesn't across the ABI boundary between C# and C++/CX.

I don't get it, since `Platform::Exception` is not a C++ native type. Is there anything special in `Platform::Exception`?

Reply:
AFAIK, there's no equivalent implementation in C# that is mapped to Platform::Exception.
So, when C# compiler found a declaration of `Platform::Exception` it will ask for the way of linking it to an implementation, which is what we are not able to provide.